### PR TITLE
feat: add configmap key `updateRequestThreshold`

### PR DESCRIPTION
**Pull request by realshuting**
## Related issue #

Add doc for https://github.com/kyverno/kyverno/pull/10382.

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

**Comment by realshuting** [comment]
This is added to 1.12.5.

File: content/en/docs/installation/customization.md [hunk 1]
```diff
@@ -276,9 +276,10 @@ The following keys are used to control the behavior of Kyverno and must be set i
...
 9. `resourceFilters`: Kubernetes resources in the format "[kind,namespace,name]" where the policy is not evaluated by the admission webhook. For example --filterKind "[Deployment, kyverno, kyverno]" --filterKind "[Deployment, kyverno, kyverno],[Events, *, *]". Note that resource filters do not apply to background scanning mode. See the [Resource Filters](#resource-filters) section for more complete information.
-10. `webhooks`: specifies the Namespace or object exclusion to configure in the webhooks managed by Kyverno. Default is `'[{"namespaceSelector": {"matchExpressions": [{"key":"kubernetes.io/metadata.name","operator":"NotIn","values":["kyverno"]}]}}]'`.
-11. `webhookAnnotations`: instructs Kyverno to add annotations to its webhooks for AKS support. Default is undefined. See the [AKS notes](platform-notes.md#notes-for-aks-users) section for details.
-12. `webhookLabels`: instructs Kyverno to add labels to its webhooks. Default is undefined.
+10. `updateRequestThreshold`: sets the threshold for the total number of updaterequests generated for mutateExisting and generate policies. The default is 1000.
```
Comment (chipzoller) [inline content/en/docs/installation/customization.md:279]: Is the value here string or integer?

Comment (realshuting) [reply content/en/docs/installation/customization.md:279]: It's a string, let me clarify it.

Comment (realshuting) [inline content/en/docs/installation/customization.md:279]:
```suggestion
10. `updateRequestThreshold`: sets the threshold for the total number of updaterequests generated for mutateExisting and generate policies. It takes the value of the string and default is 1000.
```

Comment (realshuting) [reply content/en/docs/installation/customization.md]: Updated.

**Review by chipzoller — APPROVED** [review]

**Comment by chipzoller** [comment]
Will have to manually CP if want on 1.12 branch.

**Pull request merged**